5_databases/week3/week3.sql

```diff
@@ -0,0 +1,123 @@
+USE week3_1;
+
+-- Get all meals
+SELECT *
+from meal;
+
+-- Add a new meal
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) VALUES 
+('Elephant brain', 'it will be a starter', 'North Korea', '2024-10-25 06:54:16', 8, 200.50, '2023-09-10 11:05:05');
+
+-- Get a meal with any id, fx 1
+SELECT *
+from meal
+WHERE id = 1;
+
+-- Update a meal. Update any attribute fx the title or multiple attributes
+UPDATE meal
+SET location = 'Berlin'
+WHERE price >= 200;
+
+-- Delete a meal with any id, fx 1
+DELETE FROM meal
+WHERE id = 8;
+
+-- Get all reservations
+SELECT *
+from reservation;
+
+-- Add a new reservation
+INSERT INTO reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) VALUES 
+(6, 2, '2023-09-09 10:08:07', 22042021, 'Golf', 'golf@gmail.com');
+
+-- Get a reservation with any id, fx 1
+
+SELECT *
+from reservation
+WHERE id = 1;
+
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE reservation
+SET contact_phonenumber = 206790
+WHERE contact_email like '%gmail%';
```

Comment:
Here you are asked to update a row with a specific id. 

Also, since `contact_phonenumber` is a string the value should be quoted: `SET contact_phonenumber = '206790'`